library/Icinga/Web/RememberMe.php

```diff
@@ -217,20 +223,37 @@ public function authenticate()
             $user->setDomain(Config::app()->get('authentication', 'default_domain'));
         }
 
-        $authenticated = $authChain->authenticate($user, $password);
-        if ($authenticated) {
-            $auth->setAuthenticated($user);
+        foreach (AesCrypt::METHODS as $method) {
+            $this->aesCrypt->setMethod($method);
```

Comment:
I believe we said that we don't want a fallback anymore? If `decryptFromBase64()` doesn't work, the user isn't logged in.